core/linux/common.cpp

```diff
@@ -294,7 +295,7 @@ void enable_runfast()
 }
 
 void linux_fix_personality() {
-        #if !defined(TARGET_BSD) && !defined(_ANDROID) && !defined(TARGET_OS_IPHONE) && !defined(TARGET_NACL32) && !defined(TARGET_EMSCRIPTEN)
```

Comment:
`TARGET_OS_MAC` is used now instead of `TARGET_OS_IPHONE`?

Reply:
No, `TARGET_OS_MAC` applies to OSX / iOS, while `TARGET_OS_IPHONE` is exclusively for iOS.